lib/grape/dsl/routing.rb

```diff
@@ -28,6 +28,7 @@ module ClassMethods
         #
         def version(*args, &block)
           if args.any?
+            args.flatten!
```

Comment:
Good point. I've extracted version parameter into local variable to describe the intention in a more explicit way => 82e1f21

In this case the upstream is defined by user. I've checked Grape documentation, but I've not found any examples how to properly define several versions for the endpoint. Looks like it is an undocumented feature that was supporting arrays before routing refactoring.

Should it be like that `version ['v1', 'v2']` or like that `version 'v1', 'v2'` or it is not a correct way to define a version at all?